town.js

```diff
@@ -46,7 +46,7 @@ function Town(index) {
             adjustAll();
             for (const action of totalActionList) {
                 if (towns[action.townNum].varNames.indexOf(action.varName) !== -1) {
-                    view.requestUpdate("updateRegular", {name: action.varName, index: this.index});
```

Comment:
This is based on [diff here](https://github.com/lloyd-delacroix/omsi-loops/commit/9e4e7b2a589a2eb0cb0463f3d431ecefc572231a#diff-010c2881e76abcea1b4db862a7ba4c5cf45855fc4093edbb5aacacc0bd1ba1ddL49)